src/lib/util/StaticResourceHandler.ts

```diff
@@ -40,6 +41,9 @@ export class StaticResourceHandler {
 	}
 
 	public identifyBufferType(buffer: Buffer): StaticResourceType {
+		// There's no typing available for the `is-zip` package, but we know that this method accepts a buffer and returns
+		// a boolean. So we'll just ignore the linting rule that complains about it.
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
Best practice is to use `await import()`. Doing this would require us to make some changes to our `tsconfig.json` that I'm a bit nervous to make in this timeframe, and this way technically works, so I'm just ignoring the linting error.